src/uu/pinky/src/pinky.rs

```diff
@@ -5,19 +5,38 @@
 
 // spell-checker:ignore (ToDO) BUFSIZE gecos fullname, mesg iobuf
 
+#[cfg(not(target_os = "openbsd"))]
```

Comment:
not a fan of this long list of openbsd exception either 
it is possible to have pinky-openbsd.rs and switch in the cargo.toml instead ?

Reply:
I'm not a fan either of these duplicate cfg exceptions for `target_os="openbsd"`.
For the 4 unsupported tools (`pinky`, `who`, `users`, `uptime`), I propose to create an alternate implementation `<tool>_openbsd.rs` (with comments for explanation) and use it via conditional compilation in `Cargo.toml`.

Do you agree before I make my changes ?

Reply:
let's do that :)

Reply:
I have a problem to implement this solution : conditional compilation with `cfg` in not supported for `lib` in `Cargo.toml` (see https://github.com/rust-lang/cargo/issues/12260).

For example, for `uptime` tool, I have:
- original file `src/uu/uptime/src/uptime.rs`
- alternate implementation for OpenBSD in `src/uu/uptime/src/uptime_openbsd.rs`
- `lib` defined in `Cargo.toml` as:
```
[lib]
path = "src/uptime.rs"
```

How to build `uptime` with `src/uu/uptime/Cargo.toml` using OpenBSD implementation for `lib` ?

Reply:
@tertsdiepraam any idea how to do it? :)

Reply:
With a little help from StackOverflow, I found a solution :)

For example, for `uptime` tool (sources in `src/uu/uptime/`):
- keep original implementation `src/uptime.rs`
- alternate implementation for OpenBSD in `src/uptime_openbsd.rs` (tool "unsupported")
- in `Cargo.toml`, rename `uptime.rs` to `lib.rs` in `lib` section

- `lib.rs`:
```rust
#[cfg(not(target_os = "openbsd"))]
mod uptime;
#[cfg(not(target_os = "openbsd"))]
pub use uptime::*;

#[cfg(target_os = "openbsd")]
mod uptime_openbsd;
#[cfg(target_os = "openbsd")]
pub use uptime_openbsd::*;
```

Are you OK with this solution before I push the corresponding modifications in this PR ?

Reply:
parfait, ship it ! :)

Reply:
@tertsdiepraam Thanks for the tip with `platform` directory for alternate implementation for different OS (as in `src/uu/whoami/src/platform/`). Some functions (`uu_app`) and variables (`options`) must be shared between each OS.

I have modified my code for `pinky`, `who`, `users` and `uptime` tools to use `platform` directory and specific implementation for OpenBSD.

I am checking my builds and tests on OpenBSD and Linux before submitting a new version of my PR.

Reply:
**New version of my PR** pushed (`git push -f`) to add OpenBSD specific implementation for pinky, users, uptime and who utils.

Thanks for the review and merge if OK.